### ISSUE TITLE
[0.14.0] test_sample_closed_starts_at_min_anomaly_if_in_range fails in CI test

### ISSUE THREAD

**Issue by olebole** [poliastro/poliastro#962]
🐞 **Problem**

When running the CI test during the build, I get the following failure:

```
_____________ test_sample_closed_starts_at_min_anomaly_if_in_range _____________

>   ???

tests/tests_twobody/test_sampling.py:36: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/tests_twobody/test_sampling.py:46: in test_sample_closed_starts_at_min_anomaly_if_in_range
    assert_quantity_allclose(result[0], min_nu, atol=1e-15 * u.rad)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

actual = <Quantity -3.14159265 rad>, desired = <Quantity 3.14159265 rad>
rtol = 1e-07, atol = <Quantity 1.e-15 rad>, kwargs = {}
np = <module 'numpy' from '/usr/lib/python3/dist-packages/numpy/__init__.py'>
_unquantify_allclose_arguments = <function _unquantify_allclose_arguments at 0x7f2ea06605e0>

    def assert_quantity_allclose(actual, desired, rtol=1.e-7, atol=None,
                                 **kwargs):
        """
        Raise an assertion if two objects are not equal up to desired tolerance.
    
        This is a :class:`~astropy.units.Quantity`-aware version of
        :func:`numpy.testing.assert_allclose`.
        """
        import numpy as np
        from astropy.units.quantity import _unquantify_allclose_arguments
>       np.testing.assert_allclose(*_unquantify_allclose_arguments(
            actual, desired, rtol, atol), **kwargs)
E       AssertionError: 
E       Not equal to tolerance rtol=1e-07, atol=1e-15
E       
E       Mismatched elements: 1 / 1 (100%)
E       Max absolute difference: 6.28318531
E       Max relative difference: 2.
E        x: array(-3.141593)
E        y: array(3.141593)

/usr/lib/python3/dist-packages/astropy/tests/helper.py:482: AssertionError
---------------------------------- Hypothesis ----------------------------------
Falsifying example: test_sample_closed_starts_at_min_anomaly_if_in_range(min_nu=<Quantity 3.14159265 rad>, ecc=<Quantity 0.>, max_nu=None)
_ test_sample_closed_starts_and_ends_at_min_anomaly_if_in_range_and_no_max_given _
```
and
```
>   ???

tests/tests_twobody/test_sampling.py:50: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/tests_twobody/test_sampling.py:63: in test_sample_closed_starts_and_ends_at_min_anomaly_if_in_range_and_no_max_given
    assert_quantity_allclose(result[0], min_nu)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

actual = <Quantity 8.8817842e-16 rad>, desired = <Quantity 6.975737e-16 rad>
rtol = 1e-07, atol = None, kwargs = {}
np = <module 'numpy' from '/usr/lib/python3/dist-packages/numpy/__init__.py'>
_unquantify_allclose_arguments = <function _unquantify_allclose_arguments at 0x7f2ea06605e0>

    def assert_quantity_allclose(actual, desired, rtol=1.e-7, atol=None,
                                 **kwargs):
        """
        Raise an assertion if two objects are not equal up to desired tolerance.
    
        This is a :class:`~astropy.units.Quantity`-aware version of
        :func:`numpy.testing.assert_allclose`.
        """
        import numpy as np
        from astropy.units.quantity import _unquantify_allclose_arguments
>       np.testing.assert_allclose(*_unquantify_allclose_arguments(
            actual, desired, rtol, atol), **kwargs)
E       AssertionError: 
E       Not equal to tolerance rtol=1e-07, atol=0
E       
E       Mismatched elements: 1 / 1 (100%)
E       Max absolute difference: 1.9060472e-16
E       Max relative difference: 0.27323954
E        x: array(8.881784e-16)
E        y: array(6.975737e-16)

/usr/lib/python3/dist-packages/astropy/tests/helper.py:482: AssertionError
---------------------------------- Hypothesis ----------------------------------
Falsifying example: test_sample_closed_starts_and_ends_at_min_anomaly_if_in_range_and_no_max_given(min_nu=<Quantity 6.975737e-16 rad>, ecc=<Quantity 0.>)

```

🖥 **Please paste the output of following commands**

Since this is not pip or conda, here is just the output of the top of the test output:
```
platform linux -- Python 3.8.3, pytest-4.6.9, py-1.8.1, pluggy-0.13.0
hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('/build/poliastro-0.14.0/.pybuild/cpython3_3.8/build/.hypothesis/examples')
rootdir: /build/poliastro-0.14.0, inifile: setup.cfg
plugins: hypothesis-4.36.2, remotedata-0.3.2, openfiles-0.5.0, astropy-header-0.1.2, doctestplus-0.6.1, arraydiff-0.3
```
Versions:
 * astropy 4.0.1+post1
 * astroquery 0.4
 * scipy 1.4.1
 * numpy 1.18.4
 * no numba installed

**Comment by astrojuanlu** [comment]
I'm so happy we're using Hypothesis now :) These are genuine bugs.

**Comment by astrojuanlu** [comment]
Fixing the first bug as part of #972. The second one was fixed in d147832e3, after the release.
